### DISCUSSION TITLE
Adapt color of stats by class

### DISCUSSION THREAD

**Discussion by Damn3d** [spicyboys/drg-completionist#142] [Ideas]
Hello!

Thanks a lot for this amazing website!

I suggest to modify the color of main page stats by color class. Exactly as you did when entering in a precise class menu! :)

![2023-08-17_205910](https://github.com/spicyboys/drg-completionist/assets/22372482/acb83ebd-ab53-4aaa-b0ac-24f60cfcfac9)

Exemple with Scout below:
![2023-08-17_210043](https://github.com/spicyboys/drg-completionist/assets/22372482/b951b17d-515b-487c-a90a-0dccb4518640)

**Comment by rob0rt** [comment]
I think this is a good suggestion. Our "overview" page is new and mostly thrown together so making it more aesthetically pleasing is certainly something we'd like to do. If you are so inclined, PRs are always welcomed! Otherwise we'll likely address this as we continue to add to and improve this page.

**Comment by rob0rt** [comment]
I decided to keep the individual cards as green/ blue but made the miner's overall progress bars their respective colors which I think looks pretty clean.

**Comment by Damn3d** [comment]
Well, it's your website, it's your choice ;)
I think you can use always primary color like yellow/lighr or dark yellow, red/light or dark red etc... for testing but as your wish :)
